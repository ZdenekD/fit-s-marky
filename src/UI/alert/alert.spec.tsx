import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Alert from '.';

describe('Alert', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Alert isOpen={true}>message</Alert>);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-alert');

        expect(component.exists()).toBe(true);
    });
});
