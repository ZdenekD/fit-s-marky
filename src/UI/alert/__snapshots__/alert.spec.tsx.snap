// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert match snapshot 1`] = `
<div class="backdrop"
     style="opacity:0"
>
</div>
<div class="alert  "
     data-test="component-alert"
     style="opacity:0;transform:scale(0.75) translateZ(0)"
>
  <button type="button"
          class="button  button"
          data-test="component-button"
  >
    Ã—
  </button>
  message
</div>
`;
